roofit/roofitcore/inc/RooFit/TestStatistics/LikelihoodGradientWrapper.h

```diff
@@ -82,6 +93,8 @@ class LikelihoodGradientWrapper {
    std::shared_ptr<RooAbsL> likelihood_;
    RooMinimizer *minimizer_;
    std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean_;
+   std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> component_offsets_;
+   std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> component_offsets_save_;
```

Comment:
```suggestion
public:
   using OffsetVec = std::vector<ROOT::Math::KahanSum<double>>;

protected:
   std::shared_ptr<OffsetVec> component_offsets_;
   std::shared_ptr<OffsetVec> component_offsets_save_;
```
That's a very long type... Can this maybe be aliased? Then it would also be more bearable to use `std::make_shared<OffsetVec>()` as a default parameter value.

Reply:
Yes, I will make this more readable.

Reply:
I made an encapsulating class `SharedOffset` that manages the two `shared_ptr`s and also has your suggestion of `OffsetVec` alias, which in turn is also used elsewhere to make the code more readable everywhere.